### ISSUE TITLE
Colorize Tabs - Feature Request

### ISSUE THREAD

**Issue by StephenRynas** [linuxmint/nemo#3158]
```
* Feature Request
 * Nemo version - 5.6.1
 * Is issue with desktop  - Desktop
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) - Mint 21.1 Cinnamon
 * Graphics hardware  - NVIDIA Corporation  GeForce GT 630 Rev. 2
 * 64 bit - 64 Bit
 ```

**Issue**
Feature Request. Nemo has added tabs.  Great feature! Each tab needs to be colorized to a different color or differentiated by gray scale. The reason for this is too visually differentiate each open tab. As it is now, it is too easy to make mistakes as the tabs appear identical.

**Steps to reproduce**

**Expected behaviour**

**Other information**

**Comment by StephenRynas** [comment]
Sample of Expected Behavior
[Tab.Sample.pdf](https://github.com/linuxmint/nemo/files/10387856/Tab.Sample.pdf)

**Comment by Jeremy7701** [comment]
Which colours for tabs in a dual pane setup?

**Comment by StephenRynas** [comment]
No preference, leave it up to the discretion of the Nemo team. The team must also consider that some people are color blind. That may mean that the user gets to decide whether the tabs should be in color or in gray scale.  The colors or gray scale can simply alternate between two "colors". The point being that each tab is clearly different from the adjacent tab. Also each tab "title" can be in a big bold font to further differentiate each tab. Attached is an alternative sample. I have really liked the tab feature.
[Alternative.Tab.Sample.pdf](https://github.com/linuxmint/nemo/files/10388354/Alternative.Tab.Sample.pdf)

**Comment by StephenRynas** [comment]
After sleeping on this overnight, I came-up with might be a better approach. See the attached sample document. Only the tab itself is colorized. The columns under the tab have a simple gray scale differentiation. As an additional thought, if there is only one tab, no colorization or grayscale would be needed.
[Alternative02.Tab.Sample.pdf](https://github.com/linuxmint/nemo/files/10394238/Alternative02.Tab.Sample.pdf)

**Comment by Jeremy7701** [comment]
Works in List view, but not in icon view.
Note that in dual pane mode, there is already a rather slight grey scale differentiation between the active/inactive panes.

**Comment by StephenRynas** [comment]
You are correct. However, I have poor vision. I find myself not really noticing the slight grey scale differentiation. I made some mistake when moving files from one tab to another. Have others had problems? So it is up to you guys to determine if greater differentiation would be needed.

**Comment by Drugwash2** [comment]
> Note that in dual pane mode, there is already a rather slight grey scale differentiation between the active/inactive panes.


> I find myself not really noticing the slight grey scale differentiation.

That is solely a matter of theming. Each theme has - or should have - specific colors/shades for active/inactive tabs.
However, there is no sane, usable theme editor that I know of which could allow the user a quick and thorough personalization of a theme. One would have to do this manually, after guessing - rather than anything - what each theme element is called and how to change its properties.

**Comment by StephenRynas** [comment]
An additional thought. Mint highlights the active workspace (in my case, with a thin blue line). Maybe this concept could be extended to the header of the active tab can be outlined in a color so that it is more easily discerned.
